pages/api/uploadAttachments.ts

```diff
@@ -0,0 +1,56 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { promises as fs } from 'fs'
+import path from 'path'
+import formidable, { File } from 'formidable'
+
+export const config = {
```

Comment:
idk what this does without reading PR - please leave comment here

Reply:
This is a configuration we need in order to allow the attachments to be sent over so it's not parsed as json I think